salesagility/SuiteCRM
# FIX #7971 - Textarea in EditView overlaps other fields

## Description




Read #7971

### Fixed Textarea in EditViews

![TextareaEditView](https://user-images.githubusercontent.com/939888/66235174-e071a200-e6c5-11e9-93ea-5c0a0478c1ac.gif)




### Fixed Textarea in QuickCreate

![TextareaQuickCreate](https://user-images.githubusercontent.com/939888/66235209-f1baae80-e6c5-11e9-8e94-b0dda69f9376.gif)



## Motivation and Context


Improve SuiteCRM UI

## How To Test This


1. Open an EditView (for example Contacts) and a QuickCreate (for example Activities in Contacts) with a Textarea
2. Drag the Textarea and verify you cannot resize it horizontally.


## Types of changes


### Final checklist
